### PR TITLE
Task 5(6)

### PR DESCRIPTION
**Pull request by tribaeka**
*! I forgot change commit prefix from task4(5) to task5(6).
keep that in mind

### CODE REVIEW

File: app/core/global-styles/inputs.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.login-form-input {
+  width: 98%;
+  margin-top: -0.5rem;
+  margin-bottom: 0.5rem;
+  padding: 0.25rem;
+  height: 1.5rem;
+  border: 0.2rem solid $grayLightColor;
+  border-radius: 0.25rem;
```
Comment (kirillparamonov) [inline app/core/global-styles/inputs.scss]: Please take a look at this file and move duplicated rules to mixins.

Comment (tribaeka) [reply app/core/global-styles/inputs.scss]: added new mixin

File: app/core/component/forms/add-course-form/add-course-form.controller.js [hunk 2]
```diff
@@ -21,18 +15,12 @@ angular
                 duration: $ctrl.courseDuration,
                 creationDate: new Date().toISOString()
             };
+
             if (course.duration !== undefined
                 && course.title !== undefined
                 && course.description !== undefined) {
                 courseService.addCourse(course);
```
[the remark this answers is not in the document]
Comment (tribaeka) [reply app/core/component/forms/add-course-form/add-course-form.controller.js]: fixed

File: app/core/component/forms/add-course-form/add-course-form.controller.js [hunk 3]
```diff
@@ -2,16 +2,10 @@
 
 angular
     .module('addCourseForm')
-    .controller('AddCourseFormController', function($rootScope, courseService, eventsFactory) {
+    .controller('AddCourseFormController', [ 'courseService', 'navigationService', '$scope', function(courseService, navigationService) {
```
[the remark this answers is not in the document]
Comment (tribaeka) [reply app/core/component/forms/add-course-form/add-course-form.controller.js]: fixed

File: app/core/component/course-list/courses-list.controller.js [hunk 4]
```diff
@@ -2,35 +2,37 @@
...
+                courseService.loadCourses()
+                    .then(function() {
+                        $ctrl.courses = courseService.getCourses();
+                        $ctrl.coursesIsLoaded = true;
+                    });
+            } else {
+                $ctrl.courses = courseService.getCourses();
+                $ctrl.coursesIsLoaded = true;
+            }
+
```
[the remark this answers is not in the document]
Comment (tribaeka) [reply app/core/component/course-list/courses-list.controller.js]: fixed

**Pull request merged**